admin/partials/sections/mfn-wp-plugin-admin-status.php

```diff
@@ -13,7 +13,29 @@
 $subscription_id = $subscription['subscription_id'] ?? "";
 $subscription_msg = $subscription_id;
 
-if ($subscription_id == "") {
+function mfn_verify_hub_subscription($subscription_id)
+{
+    $hub_url = mfn_fetch_hub_url();
+
+    if (mfn_starts_with($hub_url, "http")) {
+
+        $response = wp_remote_get($hub_url . '/verify/' . $subscription_id . "/status");
+        $content = wp_remote_retrieve_body($response);
+
+        if (strstr($content, 'subscription could not be verified')) {
```

Comment:
i'd say we should maybe flip the logic here: find a  way to discriminate a success msg sent by us, otherwise return err.

as this could potentially be called to someone elses server/endpoint which returns content without the error string.